code/_onclick/hud/monkey.dm

```diff
@@ -127,6 +127,14 @@
 	using.layer = 19
 	src.adding += using
 
+	using = getFromPool(/obj/screen)
+	using.name = "resist"
+	using.icon = ui_style
+	using.icon_state = "act_resist"
+	using.screen_loc = ui_pull_resist
+	using.layer = 19
```

Comment:
It's exactly like that on human.dm and everything looks exactly the same so I'm not sure why what